python/analysis/analysis_auto.sip

```diff
@@ -22,7 +22,6 @@
 %Include auto_generated/raster/qgsalignraster.sip
 %Include auto_generated/raster/qgsaspectfilter.sip
 %Include auto_generated/raster/qgsderivativefilter.sip
-%Include auto_generated/raster/qgsexiftools.sip
```

Comment:
You'll need to add some python code to copy QgsExifTools to the qgis.analysis library too, to avoid breaking existing code